guides/doc-Provisioning_Guide/topics/app_create_images.adoc

```diff
@@ -39,7 +39,7 @@ If you create the image on a remote server, connect to the server from your work
 * A {RHEL} 7 or 6 ISO file (see
 https://access.redhat.com/downloads/content/69/ver=/rhel---7/7.4/x86_64/product-software[{RHEL} 7.4 Binary DVD] or https://access.redhat.com/downloads/content/69/ver=/rhel---6/6.9/x86_64/product-software[{RHEL} 6.9 Binary DVD]).
 
-For more information about installing a {RHEL} Workstation, see https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/Installation_Guide/[{RHEL} 7 Installation Guide].
+For more information about installing a {RHEL} Workstation, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html-single/installation_guide/[{RHEL} 7 Installation Guide].
 
```

Comment:
were these link changes intentional?

Reply:
@jlsherrill yes, these fix 404s

Reply:
i still see these

Reply:
master is also failing on it. Can we fix it in a separate PR?